### DISCUSSION TITLE
Add a custom font

### DISCUSSION THREAD

**Discussion by M-Farih** [obytes/react-native-template-obytes#340] [Q&A]
My custom font doesn't load without using the `useFont()` hook from `expo-font`, but your documentation says:
- You only need to put the font file in the `assets/fonts` directory and update the Expo config by adding the exact file path to the config.
- Next, make sure to add your new font to the Tailwind CSS config to use it with `className`.

so, what is wrong ??

**Comment by yjose** [comment]
@M-Farih after you add your font to `assets/fonts` you need to make sure your run prebuild
